middlewares/error_pages.go

```diff
@@ -0,0 +1,69 @@
+package middlewares
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/containous/traefik/types"
+	"github.com/vulcand/oxy/forward"
+	"github.com/vulcand/oxy/utils"
+)
+
+//ErrorPagesHandler is a middleware that provides the custom error pages
+type ErrorPagesHandler struct {
+	HTTPCodeRanges     [][]int
```

Comment:
Can we restrict the inner slice to be an array of 2 elements? We'll never need more anyway.